src/lib/manager_impl.cpp

```diff
@@ -388,9 +387,12 @@ void mesg_state_mgr::make_repl_ctx(grpc_server* server, std::shared_ptr< mesg_fa
 nuraft::cb_func::ReturnCode mesg_state_mgr::internal_raft_event_handler(group_id_t const& group_id,
                                                                         nuraft::cb_func::Type type,
                                                                         nuraft::cb_func::Param* param) {
-    if (auto const [handled, ret] = handle_raft_event(type, param); handled) { return ret; }
-    if (auto sp = m_manager.lock(); sp) { return sp->generic_raft_event_handler(group_id, type, param); }
-    return nuraft::cb_func::Ok;
+    // Have we shutdown?
+    if (auto sp = m_manager.lock(); sp)
+        sp->generic_raft_event_handler(group_id, type, param);
+    else
+        return nuraft::cb_func::ReturnNull;
```

Comment:
NIT: pls add a log here(before line 394) to print out the ManagerImpl instance is destroyed(or shutdown), which will be helpful for debug